docs/spacer/react.tsx

```diff
@@ -17,10 +17,10 @@ interface Node {
 
 type NodeChildren = string | Node | Node[];
 
-function hasSpacer(node: Node): boolean {
-  if (node.type?.displayName === (Spacer as FC).displayName) return true;
+function hasSpacer(node?: Node): boolean {
```

Comment:
The change here is to allow conditional rendering in stories, like `{someCondition && <some-element />}`.

Reply:
We actually never use `<Spacer>` inside a story, we can later simplify this decorator.

Reply:
We actually never use <Spacer> inside a story, we can later simplify this decorator.